homeassistant/components/advantage_air/entity.py

```diff
@@ -36,19 +36,19 @@ def __init__(self, instance, ac_key):
         )
 
     @property
-    def _ac(self):
+    def _ac(self) -> dict:
         return self.coordinator.data["aircons"][self.ac_key]["info"]
 
 
 class AdvantageAirZoneEntity(AdvantageAirAcEntity):
     """Parent class for Advantage Air Zone Entities."""
 
-    def __init__(self, instance, ac_key, zone_key):
+    def __init__(self, instance, ac_key: str, zone_key: str) -> None:
         """Initialize common aspects of an Advantage Air zone entity."""
         super().__init__(instance, ac_key)
-        self.zone_key = zone_key
+        self.zone_key: str = zone_key
         self._attr_unique_id += f"-{zone_key}"
 
     @property
-    def _zone(self):
+    def _zone(self) -> dict:
```

Comment:
Can the keys be something other than strings?
```suggestion
    def _zone(self) -> dict[str, Any]:
```